### PR TITLE
perf: move out non-reused components

### PR DESCRIPTION
**Pull request by frenautvh**
| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #... 
| License          | BSD 3-Clause

## Description

Some modules in cloud-universe-components are used only once. Move them out to the corresponding app or module directly. This helps to avoid bundling unnecessary components.

(for example d3 has been removed from cloud-universe-components and won't be bundled anymore on manager dedicated app)

⚠️ billing/consumption section has been removed from cloud application as i suppose it's dead code, i would like some confirmation to ensure that i'm not missing something here

Note regarding adding breaking change or not to the commit :
I think it's debatable since cloud-universe-components is a private package, semver is basically ignored because at each build the last version will be used. Making the commit a breaking change would also major bump some apps & modules which i think is not what we want (and splitting in 2 commits seems difficult since files are moved and not created/deleted). Please share your point of view if you disagree thanks

### CODE REVIEW

File: packages/manager/modules/dbaas-logs/src/index.js [hunk 1]
```diff
@@ -8,11 +8,13 @@ import '@ovh-ux/ng-ovh-doc-url';
 import { Environment } from '@ovh-ux/manager-config';
 
 import logs from './logs';
+import cuiDualList from './components/dual-list';
```
Comment (marie-j) [inline packages/manager/modules/dbaas-logs/src/index.js:11]: I'm wondering if we can replace all those `cui` components by `ovh-ui-kit` ones as a next step

File: packages/manager/modules/pci/src/components/index.js [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
 import runabove from './runabove';
 import writeRightRequired from './writeRightRequired';
 
 const moduleName = 'ovhManagerPciComponents';
 
-angular.module(moduleName, [project, runabove, writeRightRequired]);
+angular.module(moduleName, [
+  cucConsumption,
+  cucCurrency,
+  cucFlavor,
```
Comment (marie-j) [inline packages/manager/modules/pci/src/components/index.js:15]: If I have a look this is only used for one function of one service across all manager, maybe we can just replace its usage

**Review by marie-j — COMMENTED** [review]
Regarding commit history, it would be better to have a BREAKING CHANGE commit but I agree that it doesn't make sense for other modules than `cloud-universe-components`

File: packages/manager/modules/cloud-universe-components/src/index.js [hunk 3]
```diff
@@ -1,30 +1,21 @@
...
-import cucMonitoring from './monitoring';
 import cucNavigation from './navigation';
 import cucOrderedHash from './orderedHash';
 import cucPoll from './poll';
 import cucPrice from './price';
 import cucRegion from './region';
 import cucProducts from './products';
-import cucSmoothScrollHere from './smoothScrollHere';
-import cucSpaceMeter from './space-meter';
 import cucSshKeyMin from './sshKeyMin';
```
Comment (radireddy) [inline packages/manager/modules/cloud-universe-components/src/index.js:19]: Looks like there is the same filter in the dedicated app.  We have moved it to dedicated universe components.

https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/account/user/components/filters/sshkeyMin/sshkeyMin.js

File: packages/manager/modules/pci/src/components/index.js (hunk 2 above)
Comment (frenautvh) [reply packages/manager/modules/pci/src/components/index.js]: i just deleted cucFlavor module and copied the "addOverQuotasInfos" where it's used (in kubernetes node-pool nodes add)

File: packages/manager/modules/cloud-universe-components/src/index.js (hunk 3 above)
Comment (frenautvh) [reply packages/manager/modules/cloud-universe-components/src/index.js:19]: yes but it's also used in cloud app, how did you handle the case ?
(https://github.com/ovh/manager/blob/master/packages/manager/apps/cloud/client/app/cloud/project/compute/ssh/cloud-project-compute-ssh.html#L331)

Comment (radireddy) [reply packages/manager/modules/cloud-universe-components/src/index.js:19]: I moved it to dedicated universe componets. We need to revisit to consolidate this.

**Review by radireddy — DISMISSED** [review]

**Review by marie-j — DISMISSED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by cbourgois — DISMISSED** [review]

**Comment by cbourgois** [comment]
Hey @frenautvh there is some conflicts (maybe related with #4306)

**Review by antleblanc — CHANGES_REQUESTED** [review]
Well done!

Only noticed two translations files that can be removed.

File: packages/manager/modules/nasha/src/components/space-meter/translations/Messages_lt_LT.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
```
[the remark this answers is not in the document]
Comment (frenautvh) [reply packages/manager/modules/nasha/src/components/space-meter/translations/Messages_lt_LT.json]: thanks for spotting it, i deleted the two unused translation files

**Review by antleblanc — APPROVED** [review]

**Pull request merged**